keyboards/contra/keymaps/enigma/keymap.c

```diff
@@ -0,0 +1,708 @@
+              normal_keycode = KC_P;
+              break;
+          case EN_Q:
+              letter = 'Q';
+              normal_keycode = KC_Q;
+              break;
+          case EN_R:
+              letter = 'R';
+              normal_keycode = KC_R;
+              break;
+          case EN_S:
+              letter = 'S';
+              normal_keycode = KC_S;
+              break;
+          case EN_T:
+              letter = 'T';
+              normal_keycode = KC_T;
+              break;
+          case EN_U:
+              letter = 'U';
+              normal_keycode = KC_U;
+              break;
+          case EN_V:
+              letter = 'V';
+              normal_keycode = KC_V;
+              break;
+          case EN_W:
+              letter = 'W';
+              normal_keycode = KC_W;
+              break;
+          case EN_X:
+              letter = 'X';
+              normal_keycode = KC_X;
+              break;
+          case EN_Y:
+              letter = 'Y';
+              normal_keycode = KC_Y;
+              break;
+          case EN_Z:
+              letter = 'Z';
+              normal_keycode = KC_Z;
+              break;
+          case EN_RES:
+              copy_settings(&default_settings, &current_settings);
+              break;
+          case EN_TEST:
+              for (int i = 0; i < 1000; i++) {
+                  send_character(encipher('A', &current_settings) - ('A' - 'a'));
+              }
+              break;
+          case EN_DIAG:
+              send_character(current_settings.reflector);
+              send_string(". ");
+              send_string(rotor_name(current_settings.rotor_order[0]));
+              send_character(current_settings.rotor_rings[0]);
+              send_string("-");
+              send_string(rotor_name(current_settings.rotor_order[1]));
+              send_character(current_settings.rotor_rings[1]);
+              send_string("-");
+              send_string(rotor_name(current_settings.rotor_order[2]));
+              send_character(current_settings.rotor_rings[2]);
+              send_string(" (");
+              send_character(current_settings.rotor_positions[0]);
+              send_string(", ");
+              send_character(current_settings.rotor_positions[1]);
+              send_string(", ");
+              send_character(current_settings.rotor_positions[2]);
+              send_string(") ");
+              for (int i = 0; i < current_settings.plug_count; i++) {
+                  send_character(current_settings.plugs[i * 2]);
+                  send_character(current_settings.plugs[i * 2 + 1]);
+                  send_string(" ");
+              }
+              send_string("\n");
+              break;
+          case EN_BKSP:
+              rotors_reverse(&current_settings);
+              tap_code(KC_BSPC);
+              break;
+      }
+    }
+    int mods = get_mods();
+    int any_mods = (mods & MOD_MASK_CTRL) || (mods & MOD_MASK_ALT) || (mods & MOD_MASK_GUI);
```

Comment:
get_mods is a "uint8_t" type
```suggestion
    uint8_t mods = get_mods();
    bool any_mods = (mods & MOD_MASK_CTRL) || (mods & MOD_MASK_ALT) || (mods & MOD_MASK_GUI);
```